### PR TITLE
fix: release note link for videos with two versions

### PR DESCRIPTION
**Pull request by mxschmitt**
When release videos were specified with two versions [here](https://github.com/microsoft/playwright.dev/blob/f591850f3decde10860ba389ce71fbfd55bb438c/src/data/release-videos.tsx#L13), the anchor link wasn't generated correctly and it was yielding to [this](https://github.com/microsoft/playwright.dev/actions/runs/9954951874/job/27501690410#step:4:771) warning / user facing UX issue.

### CODE REVIEW

File: src/components/VideoCards/index.tsx [hunk 1]
```diff
@@ -64,10 +70,7 @@ function VideoCard({ video }) {
         {video.version ? (
           <div className={styles.videoCardHeader}>
             <Link
-              href={`/docs/release-notes#version-${video.version.replace(
-                '.',
-                ''
-              )}`}
+              href={`/docs/release-notes#${videoToReleasePageAnchor(video)}`}
```
Comment (mxschmitt) [inline src/components/VideoCards/index.tsx:73]: this is the main fix, the rest is drive-by.

**Review by dgozman — APPROVED** [review]
